src/js/editor/editor.js

```diff
@@ -384,6 +386,13 @@ class Editor {
     return activeSections[activeSections.length - 1];
   }
 
+  detectRangeMarkup(range, markupTagName) {
```

Comment:
`hasMarkupInRange` implies a boolean, and this does actually return the first one found ala a `detect`